cli/command/orchestrator.go

```diff
@@ -68,8 +75,27 @@ func GetStackOrchestrator(flagValue, value string, stderr io.Writer) (Orchestrat
 	if o, err := normalize(env); o != orchestratorUnset {
 		return o, err
 	}
-	// Check specified orchestrator
-	if o, err := normalize(value); o != orchestratorUnset {
+	// if there is a current context, check context orchestrator
+	ctxName := dockerCli.CurrentContext()
+	if ctxName != "" && ctxName != ContextDockerHost {
+		rawMeta, err := dockerCli.ContextStore().GetContextMetadata(ctxName)
+		if err != nil {
+			return defaultOrchestrator, err
+		}
+		meta, err := GetContextMetadata(rawMeta)
+		if err != nil {
+			return defaultOrchestrator, err
+		}
+		if meta.StackOrchestrator != orchestratorUnset {
+			return meta.StackOrchestrator, nil
+		}
+	}
+	// Check specified orchestrator in global config file
+	configFile := dockerCli.ConfigFile()
```

Comment:
I think this part should be reverted to the previous state. I don't understand why you put this part inside?